pages/help.py

```diff
@@ -1050,7 +1050,7 @@ def get_content(self):
 things, so you may not get an instant answer: **Ask your question/say what
 the problem is, then wait for a few minutes, and somebody may help you**.
 
-[Chat with us](https://webchat.freenode.net/?randomnick=1&channels=freenet)
+[Chat with us](#chatlink "chatlink")
```

Comment:
Would you mind moving this logic out of the translation by using [reference-style links](http://daringfireball.net/projects/markdown/syntax#link)?

E.g.
```python
md(_("""

...

[Chat with us][url_chat]

...

""") + "\n\n" + """
[url_chat]: #chatlink "chatlink"
""")
```